bazel/repository_locations.bzl

```diff
@@ -492,27 +506,27 @@ DEPENDENCY_REPOSITORIES = dict(
         cpe = "cpe:2.3:a:icu-project:international_components_for_unicode",
     ),
     proxy_wasm_cpp_sdk = dict(
-        sha256 = "7d9e1f2e299215ed3e5fa8c8149740872b1100cfe3230fc639f967d9dcfd812e",
-        strip_prefix = "proxy-wasm-cpp-sdk-5cec30b448975e1fd3f4117311f0957309df5cb0",
-        urls = ["https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/archive/5cec30b448975e1fd3f4117311f0957309df5cb0.tar.gz"],
+        sha256 = "3276701b049328ce11cfb87d524ec0216b80d73a99787017a75828778301f31b",
+        strip_prefix = "proxy-wasm-cpp-sdk-68920464571265b237e002bfd23f91d0f9328975",
+        urls = ["https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/archive/68920464571265b237e002bfd23f91d0f9328975.tar.gz"],
         use_category = ["dataplane"],
         cpe = "N/A",
     ),
     proxy_wasm_cpp_host = dict(
-        sha256 = "494d3f81156b92bac640c26000497fbf3a7b1bc35f9789594280450c6e5d8129",
-        strip_prefix = "proxy-wasm-cpp-host-928db4d79ec7b90aea3ad13ea5df36dc60c9c31d",
-        urls = ["https://github.com/proxy-wasm/proxy-wasm-cpp-host/archive/928db4d79ec7b90aea3ad13ea5df36dc60c9c31d.tar.gz"],
+        sha256 = "1cc5334ce433507371d7ccb58c87e07ca6bd1582a75a5dafe02a89a54106db98",
+        strip_prefix = "proxy-wasm-cpp-host-4598dc3d0d3b731be1dcccb400732c72bebb1380",
+        urls = ["https://github.com/proxy-wasm/proxy-wasm-cpp-host/archive/4598dc3d0d3b731be1dcccb400732c72bebb1380.tar.gz"],
         use_category = ["dataplane"],
         cpe = "N/A",
     ),
     emscripten_toolchain = dict(
-        sha256 = "2bdbee6947e32ad1e03cd075b48fda493ab16157b2b0225b445222cd528e1843",
+        sha256 = "4ac0f1f3de8b3f1373d435cd7e58bd94de4146e751f099732167749a229b443b",
         patch_cmds = [
-            "./emsdk install 1.39.19-upstream",
-            "./emsdk activate --embedded 1.39.19-upstream",
+            "./emsdk install 1.39.6-upstream",
+            "./emsdk activate --embedded 1.39.6-upstream",
         ],
-        strip_prefix = "emsdk-dec8a63594753fe5f4ad3b47850bf64d66c14a4e",
-        urls = ["https://github.com/emscripten-core/emsdk/archive/dec8a63594753fe5f4ad3b47850bf64d66c14a4e.tar.gz"],
+        strip_prefix = "emsdk-1.39.6",
+        urls = ["https://github.com/emscripten-core/emsdk/archive/1.39.6.tar.gz"],
         use_category = ["build"],
     ),
```

Comment:
Latest is 1.40.1
```
    emscripten_toolchain = dict(
        sha256 = "81f620f17d81c18ab84c69ddfcabe813b2bf4250d69729b961ad57101a16c218",
        patch_cmds = [
            "./emsdk install 1.40.1-upstream",
            "./emsdk activate --embedded 1.40.1-upstream",
        ],
        strip_prefix = "emsdk-1.40.1",
        urls = ["https://github.com/emscripten-core/emsdk/archive/1.40.1.tar.gz"],
        use_category = ["build"],
    ),
```

Reply:
There is an interaction between the toolchain and the newer versions of emscripten that will require some work on the SDK and bazel rules.  Consequently  I am proposing rolling back the version until this can be sorted out.

Reply:
Maybe add a TODO?

Reply:
done.